Code/Tools/AssetProcessor/native/utilities/ApplicationManagerBase.cpp

```diff
@@ -1521,6 +1521,11 @@ bool ApplicationManagerBase::Activate()
     AssetProcessor::AssetProcessorStatusEntry entry(AssetProcessor::AssetProcessorStatus::Initializing_Builders, 0, QString());
     Q_EMIT AssetProcessorStatusChanged(entry);
 
+    // Start up a thread which will request a builder to start to handle the registration of gems/builders
+    // Builder info will be sent back to the AP via the network connection, start up will wait for the info before continuing
+    // See ApplicationManagerBase::InitConnectionManager BuilderRegistrationRequest for the resume point here
+    // Note that we can't wait here because the message comes back as a network message, which requires the main thread to process it
```

Comment:
nit: Avoid 'we' in code comments as it can be confusing if who "we" is to external contributors. Conside using "Code can't wait here because" or the "the Asset Processor"..

Reply:
Fixed